docs/qa/setup_local_qa_cops_test.rst

```diff
@@ -0,0 +1,177 @@
+
+Retrieve Project & Config File
+------------------------------
+
+1. Clone repository containing the pipeline configuration file to your local directory:
+
+.. code-block:: bash
+
+   $ git clone git@github.com:openstax/concourse-pipelines.git
+
+2. Change to bakery directory to retrieve the correct pipeline (there are several other pipelines in this repository):
+
+.. code-block:: bash
+
+   $ cd bakery
+
+3. Check to see if the pipeline.yml exists in the this directory:
+
+.. code-block:: bash
+
+   $ ls
+
+You should see pipeline.yml listed. This file is used to configure the pipeline.
+
+Setup local concourse server
+----------------------------
+
+0. Prerequisites:
+
+- `Docker <https://www.docker.com/>`_
+
+- `fly CLI - Concourse command line tool <https://concourse-ci.org/fly.html>`_
+
+- `wget <https://www.gnu.org/software/wget/>`_
+
+Steps to install wget (as per INSTALL document in the directory):
+
+- download wget-1.20.3.tar.gz (or latest)
+
+- unzip and cd into wget directory
+
+- run brew install gnutls
+
+- run ./configure
+
+- run make
+
+- run make install
+
+- run make clean
+
+1. Retrieve the docker-compose file to set up concourse server locally.
+In the /bakery directory that contains your pipeline.yml, run:
+
+.. code-block:: bash
+
+   $ wget https://concourse-ci.org/docker-compose.yml
+
+2. Start your Concourse server locally:
+
+.. code-block:: bash
+
+   $ docker-compose up -d
+
+   Terminal output should show the following:
+   Creating network "bakery_default" with the default driver
+   Creating bakery_concourse-db_1 ... done
+   Creating bakery_concourse_1  ... done
+
+3. To see the pipelines in Concourse UI at work, navigate to:
+
+   `localhost:8080 <localhost:8080>`_
+
+Setup pipeline with local Concourse and pipeline.yml
+----------------------------------------------------
+
+1. Target the Concourse UI, so you are able to set the pipelines to UI:
+
+.. code-block:: bash
+
+   $ fly -t local-pipeline-stuff login -c http://localhost:8080 -u test -p test
+
+   where local-pipeline-stuff is the pipeline name,
+   url is the concourse UI,
+   -u and -p is user and password.
+
+If you get a warning about versions being out of sync, run the provided command.
+
+2. Set the pipeline with the pipeline.yml config file:
+
+.. code-block:: bash
+
+   fly -t local-pipeline-stuff set-pipeline -p pdf-producer -c pipeline.yml
+
+You will be prompted to apply configuration? [yN]: y
+
+3. See the pipeline that was set from the command line in the UI by navigating to `localhost:8080 <localhost:8080>`_
+and login.
+
+4. You can unpause the pipeline from the UI, after creating a job in `cops.cnx.org <https://cops.cnx.org>`_
+for the pipeline to grab.
+
+Setting up local pipeline to monitor jobs on production cops (temporary solution)
+---------------------------------------------------------------------------------
+
+1. in a terminal, run:
+
+.. code-block:: bash
+
+   cd /Users/om9/Documents/Projects/concourse-pipelines/bakery
```

Comment:
might want to substitute some template text to let the user know to replace `/Users/om9/Documents/Projects/concourse-pipelines/bakery` with their own path on their local machine ex. `/path/to/bakery/repo` you can include yours as an example. if you want. Seems like you need to be more clear that you need the concourse-pipelines repo. Maybe a step where you clone it down if you need to?

Reply:
i changed this to cd into the cloned repo. Cloning of the repo is one of the first steps at the top